lib/l10nUtil.js

```diff
@@ -138,11 +136,49 @@ module.exports.ethereumRemoteClientPaths = [
   '../../../ethereum-remote-client/brave/app/_locales/en/messages.json'
 ]
 
+// Helper functions that's, for a given pair of chromium to brave GRD mapping
+// from the supplied map, determines which GRDP parts are no longer present in
+// the chromium GRD file.
+async function getRemovedGRDParts(mapping) {
+  let removedMap = new Map()
+  for (const [sourcePath, destPath] of Object.entries(mapping)) {
+    if (path.extname(destPath) === ".grd") {
+      const braveGRDPs = getGrdPartsFromGrd(destPath)
+      const chromiumGRDPs = getGrdPartsFromGrd(sourcePath)
+      let removed = new Set()
+      for (i = 0; i < braveGRDPs.length; i++) {
+        if (!chromiumGRDPs.includes(braveGRDPs[i])) {
+          removed.add(braveGRDPs[i])
+        }
+      }
+      if (removed.size) {
+        removedMap.set(destPath, removed)
+      }
+    }
+  }
+  return removedMap
+}
+
+// Helper function to pretty print removed GRDP file names.
+module.exports.logRemovedGRDParts = function (mapping) {
+  if (mapping.size) {
+    console.log("\n**************************************************************************")
+    console.log("The following GRDP files are no longer in the corresponding Chromium GRDs:\n")
```

Comment:
`console.log` _does_ include a line-break. Or are you meaning to put a blank line between logs?

Reply:
Yes, just wanted to make this section of the output a bit more noticeable:
<pre>
-----------
brave-browser\src\brave: python script/chromium-rebase-l10n.py --source_string_path brave-browser\src\brave\android\java\strings\xr_consent_ui_strings_java.grdp
Rebasing source string file: brave-browser\src\brave\android\java\strings\xr_consent_ui_strings_java.grdp
filename: xr_consent_ui_strings_java
writing file  brave-browser\src\brave\android\java\strings\xr_consent_ui_strings_java.grdp
-----------

**************************************************************************
The following GRDP files are no longer in the corresponding Chromium GRDs:

  From brave-browser\src\brave\components\components_strings.grd:
    - ssl_errors_strings.grdp
    - sync_ui_strings.grdp
    - translate_strings.grdp
    - undo_strings.grdp
    - version_ui_strings.grdp
</pre>